packages/onboarding/src/navigator/navigator-screens/hooks/use-navigator-screens.tsx

```diff
@@ -19,6 +22,17 @@ const useNavigatorScreens = ( screens: NavigatorScreenObject[] ) => {
 					/>
 					{ content }
 					<div className="navigator-screen__footer">
+						<NavigatorBackButton
```

Comment:
The `<` button for the mobile. The current implementation is to make the navigator footer overlap the default navigator of the step container on the mobile. The reason is the navigator screen is an individual component so it should not know the default navigator. Also, the parent component should not know the current screen as much as possible. Hence, I don't want to let the parent component change the default navigator of the step container if possible. WDYT?